mathics/builtin/arithmetic.py

```diff
@@ -1135,12 +1135,16 @@ class DirectedInfinity(SympyFunction):
     }
 
     def to_sympy(self, expr, **kwargs):
-        if len(expr.leaves) == 1:
+        if len(expr._leaves) >= 1:
```

Comment:
Why do we need ">=" instead of "=="? 

https://reference.wolfram.com/language/ref/DirectedInfinity.html indicates there is either zero or one leaves only?

Reply:
For upward compatibility, and because I was too lazy to check the documentation :)